vizro-ai/examples/dashboard_ui/app.py

```diff
@@ -0,0 +1,191 @@
+"""VizroAI UI dashboard configuration."""
+
+import json
+
+import pandas as pd
+import vizro.models as vm
+import vizro.plotly.express as px
+from actions import data_upload_action, display_filename, run_vizro_ai
+from components import CodeClipboard, Icon, MyDropdown, MyPage, OffCanvas, UserPromptTextArea, UserUpload
+from dash import Input, Output, State, callback, dcc
+from dash.exceptions import PreventUpdate
+from vizro import Vizro
+
+vm.Container.add_type("components", UserUpload)
+vm.Container.add_type("components", MyDropdown)
+vm.Container.add_type("components", OffCanvas)
+vm.Container.add_type("components", CodeClipboard)
+vm.Container.add_type("components", Icon)
+
+MyPage.add_type("components", UserPromptTextArea)
+MyPage.add_type("components", UserUpload)
+MyPage.add_type("components", MyDropdown)
+MyPage.add_type("components", OffCanvas)
+MyPage.add_type("components", CodeClipboard)
+MyPage.add_type("components", Icon)
+
+
+class CustomDashboard(vm.Dashboard):
+    """Custom Dashboard model."""
+
+    def build(self):
+        dashboard_build_obj = super().build()
+        dashboard_build_obj.children.append(dcc.Store(id="data-store-id", storage_type="local"))
+        dashboard_build_obj.children.append(dcc.Store(id="api-store-id", storage_type="local"))
```

Comment:
For data and key, why do we choose `storage_type="local"` over `session`?

Reply:
Agree, would also go for session if possible